src/Immunisatie-0-1/Cert/XIS-Server/imm-serve-1-1.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="imm-serve-1-1"/>
+    <version value="r4-1.0.0"/>
+    <name value="Immunization - XIS Server - Scenario 0.1 - Serve Immunization resources of van Putten"/>
+    <description value="Scenario 1.1 - Serve Immunization resources of van Putten"/>
+    <nts:patientTokenFixture href="imm-Patient-01-token.xml"/>
+
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url" sequenceNumber="0"/>
+
+    <test>
+        <name value="Scenario 1.1 - Serve Immunization resources"/>
+        <description
+            value="Serve Immunization resource including the patient and performer"/>
+        <nts:include value="medmij/test.xis.successfulSearch" scope="common"
+            resource="Immunization"
+            params="?_include=Immunization:patient&#38;_include=Immunization:performer"
+            responseId="immunization-search-response"/>
```

Comment:
Wordt dit responseId ergens gebruikt? Anders kun je hem net zo goed weglaten

Reply:
De responseId wordt wel gebruikt voor de read van de pharmaceuticalProduct. Specifiek in de url variabele. Deze ga ik toevoegen aan de parameter van 'variable.immunization.medication.url.xml' zodat het duidelijker is dat deze in gebruik is.